.gitpod.yml

```diff
@@ -0,0 +1,20 @@
+tasks:
+  - init: 
+      make build
+    command: |
+      gp sync-done build
+      ./prometheus --config.file=documentation/examples/prometheus.yml
+  - command: |
+      cd web/ui/react-app
+      gp sync-await build
+      unset BROWSER
+      export DANGEROUSLY_DISABLE_HOST_CHECK=true
+      yarn start
+    openMode: split-right
+ports:
+  - port: 3000
+    onOpen: open-preview
+  - port: 9090
+    onOpen: ignore
+  - port: 9091
```

Comment:
Port 9091 would be the Pushgateway component usually, but we are not starting it here. Is this entry needed?

Reply:
I'm listing it here so users don't get a notification when the port is served. Same for 9090.

Reply:
But we don't expect anything to listen on this port, right? Just not sure why we specifically include port 9091, a port which does not occur in Prometheus anywhere.

Reply:
I do not remember which process it was, but there was something that opens on port 9091. Searching your repo at least there are various mentions of that port. https://github.com/prometheus/prometheus/search?q=9091&unscoped_q=9091
But I will remove it for now, as you can always add it again later.

Reply:
Yeah, those are mostly tests or examples/docs, probably because the Prometheus Pushgateway uses port 9091, but it's a completely independent process. If we included all the ports of the other Prometheus components, we'd have to include a large list of ports :) So yeah, best to omit it here.